webservices/schemas.py

```diff
@@ -348,6 +348,7 @@ class CommitteeSearchListSchema(ApiSchema):
 FilingsSchema = make_schema(
     models.Filings,
     fields={
+        'cycle': ma.fields.Int(),
```

Comment:
You are right, I was adding it as a property before but it was not sorting so I switched it and forgot to take that out.